api/envoy/api/v2/endpoint.proto

```diff
@@ -114,4 +115,29 @@ message ClusterLoadAssignment {
 
   // Load balancing policy settings.
   Policy policy = 4;
+
+  // References endpoint groups by name.
+  repeated Egds endpoint_groups = 6;
+}
+
+// New xDS payload. The size of endpoints is fully arbitrary.
+// Also the Egds requests to Pilot can be merged into one to boost efficiency
```

Comment:
probably shouldn't mention Pilot in Envoy API

Reply:
Done, I deleted it